# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/icu4c/hotfix-gcc14.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- source/i18n/regexst.h.vanilla	2024-03-28 19:07:48.963427145 +0100
+++ source/i18n/regexst.h	2024-03-28 19:09:46.717421214 +0100
@@ -41,14 +41,14 @@
     ~RegexStaticSets();
     static void    initGlobals(UErrorCode *status);
 
-    UnicodeSet    fPropSets[URX_LAST_SET] {};      // The sets for common regex items, e.g. \s
-    Regex8BitSet  fPropSets8[URX_LAST_SET] {};     // Fast bitmap sets for latin-1 range for above.
+    UnicodeSet    fPropSets[URX_LAST_SET];      // The sets for common regex items, e.g. \s
+    Regex8BitSet  fPropSets8[URX_LAST_SET];     // Fast bitmap sets for latin-1 range for above.
 
-    UnicodeSet    fRuleSets[kRuleSet_count] {};    // Sets used while parsing regexp patterns.
-    UnicodeSet    fUnescapeCharSet {};             // Set of chars handled by unescape when
+    UnicodeSet    fRuleSets[kRuleSet_count];    // Sets used while parsing regexp patterns.
+    UnicodeSet    fUnescapeCharSet;             // Set of chars handled by unescape when
                                                    //   encountered with a \ in a pattern.
-    UnicodeSet    *fRuleDigitsAlias {};
-    UText         *fEmptyText {};                  // An empty string, to be used when a matcher
+    UnicodeSet    *fRuleDigitsAlias;
+    UText         *fEmptyText;                  // An empty string, to be used when a matcher
                                                    //   is created with no input.
 
 };
